op-conductor/health/monitor.go

```diff
@@ -150,12 +151,15 @@ func (hm *SequencerHealthMonitor) healthCheck() error {
 
 	if timeDiff > hm.rollupCfg.BlockTime && expectedBlocks > blockDiff {
 		hm.log.Error(
-			"unsafe head is not progressing as expected",
+			fmt.Sprintf("unsafe head is ahead by %d", expectedBlocks),
```

Comment:
I feel like we don't need to change this log message as it is accurate in the sense that we're checking head progression, adding more detailed fields are good.

The place we should change the message is probably the check below which it checks if unsafe head is lagging behind configured interval, we should change that message